UI/TechTreeWnd.cpp

```diff
@@ -1908,7 +1908,8 @@ TechTreeWnd::TechTreeWnd(GG::X w, GG::Y h, bool initially_hidden /*= true*/) :
     m_tech_tree_controls(0),
     m_enc_detail_panel(0),
     m_layout_panel(0),
-    m_tech_list(0)
+    m_tech_list(0),
+    m_initially_hide_tree(initially_hidden)
```

Comment:
Minor gripe, should this be something like `m_init_flag`?  The current name suggests it reflects the initial state.